AI_CONTRIBUTIONS_POLICY.md

```diff
@@ -0,0 +1,159 @@
+# 🤖 AI Generated Contributions Policy for DISCOVER Cookbook
+
+## 1️⃣ Purpose and Scope
+
+This policy sets guidelines for contributions to the DISCOVER Cookbook created using or assisted by Artificial Intelligence (AI) and generative AI tools (e.g., LLMs, code generation tools, content platforms).
+
+**This policy applies to:**
+- 👩💻 All contributors, maintainers, and reviewers of the DISCOVER Cookbook
+- 📚 All content: text, code, documentation, and more
+- 🤝 Third-party collaborators or contractors
+
+## 2️⃣ Guiding Principles
+
+### 🏳️ Authenticity & Transparency
+We expect authentic engagement. Contributors must disclose AI tool use and are fully responsible for their submissions.
+
+### 👁️ Human Oversight & Understanding
+All AI-generated content must be reviewed and understood by a human. Be prepared to explain and defend your contribution.
+
+### 🎯 Quality & Accuracy
+Every contribution—AI-assisted or not—must meet the project’s high standards for quality, accuracy, and relevance.
+
+### ⚖️ Legal & Ethical Compliance
+Contributors must ensure AI-generated content complies with laws, licenses, and ethical guidelines.
+
+## 3️⃣ Permitted Use
+
+AI tools may be used for contributions under these conditions:
+
+### 💡 Assistive AI Usage
+- Brainstorming and outlines (subsequent human revision expected)
+- ✅ Grammar/spell checks & language improvements
+- 🌍 Assistance with translations
+- 🏗️ Formatting and structural help
+
+### 👩💻 Code & Technical Content
+- AI-generated code that is tested and validated
```

Comment:
AI code needs be referenced. The license issues around AI code is not super clear and we want to know which commits are using it.

Reply:
oh nevermind, I see you show this later, suggested "tested, validated, and referenced"